src/components/Content/index.tsx

```diff
@@ -65,38 +63,69 @@ export const Content: React.VFC = () => {
+        <Route path={baseRoute.configurePatch} element={<ConfigurePatch />}>
+          <Route path={tab} element={<ConfigurePatch />} />
+          <Route path="" element={<ConfigurePatch />} />
+        </Route>
+        <Route path={routes.patch}>
+          <Route path={tab} element={<VersionPage />} />
+          <Route path="" element={<VersionPage />} />
+        </Route>
+        <Route path={baseRoute.version}>
+          <Route path={tab} element={<VersionPage />} />
+          <Route path="" element={<VersionPage />} />
+        </Route>
+        <Route path={routes.jobLogs}>
+          <Route path=":groupId" element={<JobLogs />} />
+          <Route path="" element={<JobLogs />} />
+        </Route>
+        <Route path={routes.hosts} element={<Hosts />} />
+        <Route path={routes.host} element={<Host />} />
+        <Route path={routes.myPatches} element={<MyPatches />} />
         <Route
-          exact
           path={routes.userPatchesRedirect}
-          component={UserPatchesRedirect}
+          element={<UserPatchesRedirect />}
         />
         <Route
           path={routes.projectSettingsRedirect}
-          component={ProjectSettingsRedirect}
+          element={<ProjectSettingsRedirect />}
         />
-        <Route path={routes.userPatches} component={UserPatches} />
-        <Route path={routes.taskQueue} component={TaskQueue} />
-        <Route path={routes.projectPatches} component={ProjectPatches} />
-        <Route path={routes.projectSettings} component={ProjectSettings} />
-        <Route path={routes.spawn} component={Spawn} />
-        <Route path={routes.commitQueue} component={CommitQueue} />
-        <Route path={routes.preferences} component={Preferences} />
-        <Route path={routes.commits} component={Commits} />
-        <Route path={routes.taskHistory} component={TaskHistory} />
-        <Route path={routes.variantHistory} component={VariantHistory} />
-        <Route exact path="/" component={MyPatches} />
-
-        <Route component={PageDoesNotExist} />
-      </Switch>
+        <Route path={routes.userPatches} element={<UserPatches />} />
+        <Route path={routes.taskQueue}>
+          <Route path="" element={<TaskQueue />} />
+          <Route path=":distro" element={<TaskQueue />} />
+          <Route path=":distro/:taskId" element={<TaskQueue />} />
+        </Route>
+        <Route path={routes.projectPatches} element={<ProjectPatches />} />
+        <Route path={baseRoute.projectSettings}>
+          <Route path={tab} element={<ProjectSettings />} />
+          <Route path="" element={<ProjectSettings />} />
+        </Route>
+        <Route path={routes.spawn} element={<Spawn />}>
+          <Route path={SpawnTab.Host} element={<SpawnHost />} />
+          <Route path={SpawnTab.Volume} element={<SpawnVolume />} />
+          <Route path="*" element={<Navigate to={SpawnTab.Host} replace />} />
+          <Route path="" element={<Navigate to={SpawnTab.Host} replace />} />
```

Comment:
Why does this case on the tabs but the other do not?

Reply:
I would expect Preferences to look the same way but it does not. 
We should be consistent about how we handle nested tab routes.
I vote for the old way where the components handled their tabbing and not router. Willing to discuss this!

Reply:
We figured out how to do this by including a element in the parent route and omitting the element in the child route.